ClientApp/src/app/services/labels/label.service.ts

```diff
@@ -0,0 +1,43 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { environment } from 'src/environments/environment';
+import { LabelData } from 'src/types/label'
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LabelService {
+
+  constructor(private http: HttpClient) { }
+
+  label: LabelData = { label: "", color: "#ff0000"}
+  list: LabelData[] = []
+
+  readonly baseURL = `${environment.url}Labels/`
+
+  getLabels(){
+    return this.http.get(this.baseURL)
+  }
+  
+  postLabel(){
+    return this.http.post(this.baseURL, this.label)
```

Comment:
Why do this instead of just passing the label in as an argument to the function?